packages/start-frontend/__tests__/cli.test.ts

```diff
@@ -43,11 +43,13 @@ async function executeCLI(inputs: string[], delay = 500) {
 
 describe("start-frontend", () => {
   beforeAll(() => {
+    const result = execSync(`npx tree-cli ${testDir}`).toString("utf-8");
```

Comment:
It doesn't look like this `result` is being used; is there a reason why `npx tree-cli` is being executed here?